src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Testing/TestRunner.TestRunHandler.cs

```diff
@@ -138,31 +141,83 @@ private static string CreateTestCaseReportMessage(TestRunChangedEventArgs? testR
             {
                 var messageBuilder = new StringBuilder();
                 messageBuilder.Append($"[{result.Outcome}] {result.TestCase.DisplayName}");
-                if (result.ErrorMessage != null || result.ErrorStackTrace != null)
-                {
-                    messageBuilder.AppendLine();
-                }
 
                 if (!string.IsNullOrWhiteSpace(result.ErrorMessage))
                 {
+                    messageBuilder.AppendLine();
                     messageBuilder.AppendLine(IndentString($"{LanguageServerResources.Message}:", 4));
                     messageBuilder.AppendLine(IndentString(result.ErrorMessage, 8));
                 }
 
                 if (!string.IsNullOrWhiteSpace(result.ErrorStackTrace))
                 {
+                    messageBuilder.AppendLine();
```

Comment:
I think this will append an empty line between the msg and stack trace, which IIRC is why I appended only once above.  Maybe not the biggest issue but I believe that was why it was written that way